scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -451,16 +469,17 @@ task CollateReports {
   }
 }
 
-# Print given message to stderr and return an error
-task ErrorWithMessage{
+
+task CountInputVcfs {
   input {
-    String message
+    File input_vcf_fofn
   }
   command <<<
-    >&2 echo "Error: ~{message}"
-    exit 1
+    wc -l < ~{input_vcf_fofn} > num_vcfs.txt
```

Comment:
what does the < do here?

Reply:
`<` redirects the file to be the standard input of `wc`. When `wc` reads from standard input it emits a bare(ish) number delta some whitespace that `read_int` ignores:

```
% wc -l < foo.txt
       3
%
```

But with the file as an argument, `wc` "helpfully" prints the name of the file which `read_int` does not like at all:

```
% wc -l foo.txt  
       3 foo.txt
%
```